src/tests/ftest/erasurecode/ec_ior_smoke.py

```diff
@@ -6,7 +6,7 @@
 '''
 from ior_test_base import IorTestBase
 
-class ErasureCodeIor(IorTestBase):
+class EcodIor(IorTestBase):
```

Comment:
It should be EcodeIor? missing "e" in code or it's intended?

Reply:
I think the idea here is to use exactly 4 letters. E.g. "Ecod" and "Dmvr"

Reply:
Ah got it.